src/dtos/index.ts

```diff
@@ -58,7 +59,11 @@ const DTOs: Record<string, DTOFunction<never, DTO>> = {
     replace_sm: replaceSmDTO,
     deliver_sm: deliverSmDTO,
     deliver_sm_resp: deliverSmRespDTO,
-};
+} as const satisfies Record<string, DTOFunction<never, DTO>>;
+
+export type DTOPayloadMap = {
+    [K in keyof typeof DTOs]: DTOData<ReturnType<(typeof DTOs)[K]>>
```

Comment:
Add import to response, force only response events.
```suggestion
    [K in keyof typeof DTOs & ResponseCommandName]: DTOData<ReturnType<(typeof DTOs)[K]>>
```